Formula/ice.rb

```diff
@@ -19,8 +20,12 @@ class Ice < Formula
   end
 
   depends_on "lmdb"
-  depends_on macos: :catalina
   depends_on "mcpp"
+  depends_on "openssl@3"
```

Comment:
No linkage with `openssl@3` on macOS. I think this is a Linux-only dependency. We can probably drop the revision bump.